packet/flatcar-linux/kubernetes/cl/controller.yaml.tmpl

```diff
@@ -47,6 +48,24 @@ systemd:
         [Install]
         RequiredBy=kubelet.service
         RequiredBy=etcd-member.service
+    - name: create-etcd-private-ip-env-file.service
+      # This service will extract value of private interface from the env var file `/run/metadata/flatcar`.
+      # And then assign it to the variables that which will be stored in file `/etc/kubernetes/etcd.config`,
+      # this file is sourced by etcd-member service.
```

Comment:
As we have limits on the size of the user-data, I'd consider removing this comment (not sure I would in other circumstances, though). But not sure, your call :-)

Reply:
This file is converted to json via `ct` and the comment will be deleted in the process before we get the user-data AFAIR

Reply:
Ohh, makes sense. I've seen comments stayed, but maybe they were comments inside the scripts put here (like here, that is inside inline code https://github.com/kinvolk/lokomotive-kubernetes/pull/161/files#diff-45288c4191a7d4867391fac80437bd5aR216-R219).

Sorry, my bad. Nothing to see here, move along :-P